packages/kernel-6.1/1007-blk-cgroup-Optimize-blkcg_rstat_flush.patch

```diff
@@ -0,0 +1,238 @@
+From b8ac5ec734fd3f4c18efd55ad3355723d27d534d Mon Sep 17 00:00:00 2001
```

Comment:
This was `3b8cc6298724021da845f2f9fd7dd4b6829a6817` upstream.

There are six fixes tagged for this commit:
```
9d230c099 ("blk-cgroup: Properly propagate the iostat update up the hierarchy")
d0aac2363 ("blk-cgroup: fix list corruption from reorder of WRITE ->lqueued")
6da668063 ("blk-cgroup: fix list corruption from resetting io stat")
20cb1c2fb ("blk-cgroup: Flush stats before releasing blkcg_gq")
3d2af77e3 ("blk-cgroup: Reinit blkg_iostat_set after clearing in blkcg_reset_stats()")
0416f3be5 ("blk-cgroup: don't update io stat for root cgroup")
```